Gemfile.lock

```diff
@@ -1,68 +1,101 @@
 PATH
   remote: .
   specs:
-    decanter (4.0.4)
-      actionpack (>= 4.2.10)
+    decanter (5.0.0)
+      actionpack (>= 7.1.3.2)
```

Comment:
Does this change essentially restrict the usage of Decanter v5 to apps with Rails 7 and above?